uaaextras/templates/email/body.html

```diff
@@ -8,7 +8,7 @@
         <p>Hello!</p>
         <p>
             You've been invited to join
-            <a href="https://cloud.gov">{{branding.company_name}}</a>.
```

Comment:
The value of `branding.company_name` should be changed from `Cloud Foundry` to `cloud.gov`. Is that already being taken care of elsewhere?

Reply:
Yep! These are just variables, so thats the local value. Nothing is changing here, so it will continue to say cloud.gov.